src/main/java/uk/ac/ebi/eva/evaseqcol/entities/SeqColExtendedDataEntity.java

```diff
@@ -0,0 +1,53 @@
+package uk.ac.ebi.eva.evaseqcol.entities;
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import lombok.Data;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.hibernate.annotations.TypeDefs;
+
+import uk.ac.ebi.eva.evaseqcol.utils.JSONExtData;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
+@Entity
+@TypeDefs({
+        @TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)
+})
+@Data
+@Table(name = "seqcol_extended_data")
+public class SeqColExtendedDataEntity {
+
+    @Id
+    @Column(name = "digest")
+    protected String digest; // The level 0 digest
+
+    @Type(type = "jsonb")
+    @Column(columnDefinition = "jsonb")
+    @Basic(fetch = FetchType.LAZY)
+    private JSONExtData object;
```

Comment:
Same. We should avoid `object`. Not sure what to call it since it is an object containing an array but I'm sure there is a more descriptive name.

Reply:
True, I suggest calling it `extendedSeqColData`.